contracts/Document.sol

```diff
@@ -95,6 +103,8 @@ contract Document is DocumentInterface {
         signedAt = documentSignedAt;
 
         signed = true;
+        
+        notifyEntityEvent(DOCUMENT_NOTIFIERS_KEY, DOCUMENT_SIGNED_EVENT, address(this));
```

Comment:
The way of trigger the events is different than the how trigger the events the signature contract is different a this must be equal because both of them are events (no differences between events).

I think the correct way is factor the signature and avoid create the event on the signature and move all the logic to create the event here